cmd/containers-storage/mount.go

```diff
@@ -74,6 +77,36 @@ func unmount(flags *mflag.FlagSet, action string, m storage.Store, args []string
 	return 0
 }
 
+func mounted(flags *mflag.FlagSet, action string, m storage.Store, args []string) int {
+	mes := []mountPointError{}
+	errors := false
+	for _, arg := range args {
+		mounted, err := m.Mounted(arg)
+		errText := ""
+		if err != nil {
+			errText = fmt.Sprintf("%v", err)
+			errors = true
+		}
+		if mounted {
+			fmt.Printf("%s mounted\n", arg)
+		}
```

Comment:
Non-blocker: in principle it seems... off to treat `mounted` as useful if `err` is set.